dashboard/functions/index.js

```diff
@@ -222,7 +243,7 @@ function update_device_config(message, attributes) {
     versionToUpdate: version,
     binaryData: binaryData
   };
-  const commandFolder = `devices/${deviceId}/update/config`;
+  const commandFolder = `devices/${deviceId}/update/configs`;
```

Comment:
Maybe I'm missing something, but why has this folder name changed to configs (plural)?